.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,56 @@
+name: Deploy new release to Clojars
+on:
+  workflow_dispatch:
```

Comment:
Dispatch workflows are only run on demand through _Actions > [name of workflow] > Run workflow._ Basically, this is a manual release trigger with `minor` increment as default.